tests/integration/ha_tests/test_upgrade.py

```diff
@@ -38,21 +38,25 @@
 @pytest.mark.abort_on_fail
 async def test_deploy_latest(ops_test: OpsTest) -> None:
     """Simple test to ensure that the PostgreSQL and application charms get deployed."""
-    await asyncio.gather(
-        ops_test.model.deploy(
-            DATABASE_APP_NAME,
-            num_units=3,
-            channel="16/edge",
-            trust=True,
-            config={"profile": "testing"},
-            base=CHARM_BASE_NOBLE,
-        ),
-        ops_test.model.deploy(
-            APPLICATION_NAME,
-            num_units=1,
-            channel="latest/edge",
-            base=CHARM_BASE,
-        ),
+    # TODO: rollback to ops_test.model.deploy syntax when we release to edge.
+    await ops_test.juju(
```

Comment:
This will be rolled back as soon as a revision is published to the `16/edge` channel.